packages/app/src/app/[locale]/(main)/admin/users/UsersView.tsx

```diff
@@ -196,7 +196,12 @@ export default function UsersView(): JSX.Element {
       {
         accessorKey: 'enabled',
         header: () => <Text inherit>{t('usersView.table.active')}</Text>,
-        cell: info => (info.getValue() ? <IconCheck /> : <IconX />),
+        cell: info =>
+          info.getValue() ? (
+            <IconUserCheck color="#2f9e44" />
```

Comment:
Maybe you could use mantine color variables (e.g. var(--mantine-color-red)) instead of the hardcoded hex values?